features/370-auth-ui/configuration.md

```diff
@@ -0,0 +1,3 @@
+# Configuration
+
+Auth UI shares the configuration of SSO. Both of them need to configure a set of allowed redirect URLs so they should be configured in one place.
```

Comment:
Ummm, but at least CSS are some sort of styling?

And also (as mentioned in feature issue):
- Default callback URL for login / logout?
- Specify the domain for iframe?

Reply:
Will do. Supporting default redirect URI means relaxing a required parameter to a optional parameter. I guess it is OK to deviate from the spec by making required parameter optional.

Reply:
Yes I was thinking about the required to optional parameter... but actually I'm not sure if it is a good idea, so consider it a suggestion (Sorry didn't make it clear in first message)

Reply:
I thought about it again and found that supporting default redirect URI is not very useful. An example authorize URL looks like `http://192.168.1.127:8081/_auth/_/authorize?client_id=api_key&code_challenge=code_challenge&code_challenge_method=S256&redirect_uri=REDIRECT_URI&response_type=code&scope=openid`. Even if we allow default redirect URI, the query is still very long. SSO does not support default redirect URI neither.